src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -31,7 +31,7 @@ internal static partial class Libraries
         internal const string Wldap32 = "wldap32.dll";
         internal const string Ws2_32 = "ws2_32.dll";
         internal const string Wtsapi32 = "wtsapi32.dll";
-        internal const string CompressionNative = "clrcompression.dll";
+        internal const string CompressionNative = "clrcompression";
```

Comment:
It would be nice to rename this to `System.IO.Compression.Native` (fine to do it in separate PR).

Reply:
I assume you mean renaming the actual dll ? 
Yes, might be better to do that separately - just in case it breaks something that expects the old file name (packaging, installers, things in SDK repo..)

We can also easily embed it now, similar to System.Globalization, unless there are reasons we want it as a separate dll.